omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bootstrap_validator.rb

```diff
@@ -75,10 +75,11 @@ def run!
   # backend components, allow data bootstrapping to be bypassed when
   # no chef-server-running.json is present but a secrets file is present.
   def bypass_bootstrap?
-    first_run? && secrets_exists?
+    first_run? && secrets_exists? && PrivateChef['postgresql']['external']
```

Comment:
I'm ok with this change - I gave it as a workaround in a customer issue  -  but with reservations. The `bypass_bootstrap` exists as a best a attempt to determine when someone is using a completely external back-end, and is running multiple chef servers in 'standalone` mode connected to the same backend. (So effectively, they're just front-ends, without adding a new topo).   So even with this change, it could come up with the wrong answer since it's possible to have a tiered or HA setup with external postgresql. 

A better fix - and I think I saw something like this underway in another PR - is an explicit flag saying that's how we're running chef-server, and using instead of the external flag. .